Dockerfile

```diff
@@ -6,6 +6,8 @@ WORKDIR /usr/src/app
 # set environment variables
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
+ENV PYTHONPATH "${PYTHONPATH}:/app"
+
```

Comment:
If I'm not mistaking, alembic lets you pass the path to the .ini file. Perhaps it would be better?

Reply:
The environment variable `PYTHONPATH` allows to make python modules importable. This way `alembic/env.py` can import app/config and app/db from outside the `app/` path. What do you think ?

Reply:
I get your point. However, since Docker is not the only way to run the API, either we add instructions in the doc, or we try to fix this import :thinking: